### PR TITLE
DevWs for Sinope Technologies containing containing VA4200WZ-VA4200ZB Sinope Valve

### PR DESCRIPTION
**Pull request by SmartThings**

### CODE REVIEW

**Comment by aguilbault-Sinope** [comment]
@KKlimczukS , @k-morawski
This pull request was made from the Smartthings workspace to fix the pull request #65711

File: devicetypes/sinope-technologies/va4200wz-va4200zb-sinope-valve.src/va4200wz-va4200zb-sinope-valve.groovy [hunk 1]
```diff
@@ -286,6 +284,33 @@ private int get_LOG_TRACE() {
...
+
+	def index = 1;
+	for (index; value > anglesTable[index]; index++) {
+	}
+
+	def ratioBetweenPointXandY = (levelsTable[index] - levelsTable[index - 1]) / (anglesTable[index] - anglesTable[index - 1]);
+	def angleToAdd = levelsTable[index] - (anglesTable[index] * ratioBetweenPointXandY);
+	def levelWithFactor = (ratioBetweenPointXandY * value) + angleToAdd;
+	levelValue = levelWithFactor / 1000;
+	def roundedLevelValue = Math.round(levelValue);
```
Comment (MWierzbinskaS) [inline devicetypes/sinope-technologies/va4200wz-va4200zb-sinope-valve.src/va4200wz-va4200zb-sinope-valve.groovy:304]: I think that levelValue variable is not necessary. I think that Math.round(levelWithFactor / 1000) would be more readable.

**Review by MWierzbinskaS — CHANGES_REQUESTED** [review]

**Comment by k-morawski** [comment]
Dear @vbeauregard-Sinope , could you please check @MWierzbinskaS comment? If you have any questions, please let us know. Many thanks!

**Comment by k-morawski** [comment]
Dear @vbeauregard-Sinope , this is just a friendly reminder. Could you please update your PR? Thanks

**Review by aguilbault-Sinope — APPROVED** [review]

**Comment by aguilbault-Sinope** [comment]
> Dear @vbeauregard-Sinope , this is just a friendly reminder. Could you please update your PR? Thanks

Hello, Vincent is not working with our company anymore. I'm trying to make the changes mentionned for the pull-request but I can't seem to be able to push it for permission issues. I'll work it.

**Comment by k-morawski** [comment]
> > Dear @vbeauregard-Sinope , this is just a friendly reminder. Could you please update your PR? Thanks
> 
> Hello, Vincent is not working with our company anymore. I'm trying to make the changes mentionned for the pull-request but I can't seem to be able to push it for permission issues. I'll work it.

Thank you, you can create a new pull request with your changes and add a link to it in the comments section. We will review it.

**Comment by aguilbault-Sinope** [comment]
> > > Dear @vbeauregard-Sinope , this is just a friendly reminder. Could you please update your PR? Thanks
> > 
> > 
> > Hello, Vincent is not working with our company anymore. I'm trying to make the changes mentionned for the pull-request but I can't seem to be able to push it for permission issues. I'll work it.
> 
> Thank you, you can create a new pull request with your changes and add a link to it in the comments section. We will review it.

Here is the new pull-request : https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/71614

**Comment by k-morawski** [comment]
I am closing this pull request. Connected integration request has been failed.

**Pull request closed without merging**